dev/NavigationView/NavigationView_InteractionTests/FocusBehaviorTests.cs

```diff
@@ -0,0 +1,462 @@
+                    //TODO: Update RS3 and below tab behavior to match RS4+
+                    if (!PlatformConfiguration.IsOsVersionGreaterThanOrEqual(OSVersion.Redstone4))
+                    {
+                        Log.Warning("Test is disabled because the repeater tab behavior is current different rs3 and below.");
+                        return;
+                    }
+
+                    SetNavViewWidth(ControlWidth.Wide);
+
+                    Button togglePaneButton = new Button(FindElement.ById("TogglePaneButton"));
+                    UIObject searchBox = FindElement.ByNameAndClassName("PaneAutoSuggestBox", "TextBox");
+                    UIObject firstItem = FindElement.ByName("Home");
+                    UIObject settingsItem = FindElement.ByName("Settings");
+                    togglePaneButton.SetFocus();
+                    Wait.ForIdle();
+
+                    Log.Comment("Verify that pressing tab while TogglePaneButton has focus moves to the search box");
+                    KeyboardHelper.PressKey(Key.Tab);
+                    Wait.ForIdle();
+                    Verify.IsTrue(searchBox.HasKeyboardFocus);
+
+                    Log.Comment("Verify that pressing tab while the search box has focus moves to the first menu item");
+                    KeyboardHelper.PressKey(Key.Tab);
+                    Wait.ForIdle();
+                    Verify.IsTrue(firstItem.HasKeyboardFocus);
+
+                    Log.Comment("Verify that pressing tab twice more will move focus to the settings item");
+                    KeyboardHelper.PressKey(Key.Tab, ModifierKey.None, 2);
+                    Wait.ForIdle();
+                    Verify.IsTrue(settingsItem.HasKeyboardFocus);
+
+                    // TODO: Re-enable test part and remove workaround once saving tab state is fixed
+
+                    Log.Comment("Move Focus to first item");
+                    firstItem.SetFocus();
+                    Wait.ForIdle();
+
+                    //Log.Comment("Verify that pressing SHIFT+tab twice will move focus to the first menu item");
+                    //KeyboardHelper.PressKey(Key.Tab, ModifierKey.Shift, 2);
+                    //Wait.ForIdle();
+                    //Verify.IsTrue(firstItem.HasKeyboardFocus);
+
+                    Log.Comment("Verify that pressing SHIFT+tab will move focus to the search box");
+                    KeyboardHelper.PressKey(Key.Tab, ModifierKey.Shift, 1);
+                    Wait.ForIdle();
+                    Verify.IsTrue(searchBox.HasKeyboardFocus);
+
+                    Log.Comment("Verify that pressing SHIFT+tab will move focus to the TogglePaneButton");
+                    KeyboardHelper.PressKey(Key.Tab, ModifierKey.Shift, 1);
+                    Wait.ForIdle();
+                    Verify.IsTrue(togglePaneButton.HasKeyboardFocus);
+                }
+            }
+        }
+
+        [TestMethod]
+        public void CanDoSelectionChangedOfItemTemplate()
```

Comment:
>CanDoSelectionChangedOfItemTemplate [](start = 20, length = 35)

Perhaps this was a test that covered  a crash, but it seems not very useful as written.